packages/platform-ios/src/commands/runIOS/index.js

```diff
@@ -499,5 +508,11 @@ export default {
       default: process.env.RCT_METRO_PORT || 8081,
       parse: (val: string) => Number(val),
     },
+    {
+      command: '--terminal [string]',
+      description:
+        'Launches the Metro Bundler in a new window using the specified terminal path.',
+      default: undefined,
```

Comment:
Shouldn't it be same default as in other command? `process.env...`

Reply:
Yep, I'll change it!